### PR TITLE
Update the readme to reflect the current state

### PR DESCRIPTION
**Pull request by belcherj**

### CODE REVIEW

File: client/extensions/woocommerce/README.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 Sell anything, beautifully. Now from Calypso.
 
 ## Prerequisites
 
-WooCommerce for Calypso requires a few plugins to be in place on your WordPress.org installation in order to work properly:
+When setting up the Store on WordPress.com the following plugins will be installed and activated:
 
-- [Jetpack](https://github.com/Automattic/jetpack) - powers the API behind Calypso.
+- [TaxJar - Sales Tax Automation for WooCommerce](https://wordpress.org/plugins/taxjar-simplified-taxes-for-woocommerce/) - Sales Tax Automation for WooCommerce
 - [WooCommerce](https://github.com/woocommerce/woocommerce) - at least 3.1 (use `master` for now).
```
Comment (ryelle) [inline client/extensions/woocommerce/README.md]: WC 3.1 is out, do you still recommend using master?

**Comment by belcherj** [comment]
@coderkevin can you make the call on @ryelle 's comment?  Also can one of ya'll merge after that is complete?

**Comment by justinshreve** [comment]
Yeah I agree about updating it to say 3.1 now that it is out, and it's what we are installing on these sites. I can do that & merge.

**Comment by coderkevin** [comment]
LGTM 👍

**Pull request merged**